src/gov/nist/javax/sip/parser/Parser.java

```diff
@@ -43,7 +43,7 @@
 
     protected ParseException createParseException(String exceptionString) {
         return new ParseException(
-            lexer.getBuffer() + ":" + exceptionString,
+            lexer.getBuffer().trim() + ":" + exceptionString,
```

Comment:
if parsing fails, like for example negative number value, buffer will contain new line (\n) at the end, so that's the reason for the trim